src/System.IO.FileSystem.Watcher/ref/4.0.0/System.IO.FileSystem.Watcher.depproj

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You don't need this depproj and project.json either. Eric made some changes such that now we harvest the latest stable package and grab old assets from there, so you can remove this.